projects/aca-content/src/lib/services/content-management.service.ts

```diff
@@ -85,7 +85,7 @@ interface RestoredNode {
   providedIn: 'root'
 })
 export class ContentManagementService {
-  private readonly createMenuButtonSelector = 'app-create-menu button';
+  private readonly createMenuButtonSelector = 'app-toolbar-menu button[title="Create content"]';
```

Comment:
We should avoid hard-coded strings like this, otherwise it won't be translated for users using other languages

Reply:
modified the selector to pick elements from **id** instead of the title so it will not be dependent on languages or translations.